res/values/strings.xml

```diff
@@ -95,6 +96,8 @@
     <string name="last_month">Last month</string>
     <!-- Placeholder replaced by date or time, resulting in "Last seen at 12:13 AM" or "Last seen Nov 12" -->
     <string name="last_seen_at">Last seen at %1$s</string>
+    <!-- used by desktop, momentjs provides the relative time in local format like: "3 minutes ago" in english or "vor 3 Minuten" in german.-->
+    <string name="last_seen_relative">Last seen %1$s</string>
```

Comment:
i think, the concatenation is okayish here - it is not just concatenation, but the translators could also do things as `Last seen: %1$s` if grammar does not work otherwise.  
(the "Member X %1$s by %2$s" was different as there was really a concatenation without giving the locales any control)

issue may be that the replacement is not under the control of the translators, that may make things harder. however, let's be pragmatic and see if that really becomes an issue and act then. but i would change the hint to be more on-point:

```suggestion
    <!-- The placeholder will be replaced by a relative point in time as "3 minutes ago" (see https://momentjs.com for more examples and languages)-->
    <string name="last_seen_relative">Last seen %1$s</string>
```

Reply:
> issue may be that the replacement is not under the control of the translators, that may make things harder. however, let's be pragmatic and see if that really becomes an issue and act then. but i would change the hint to be more on-point:

If there are languages that momentjs does not support we can add them https://momentjs.com/docs/#/i18n/changing-locale/